app/full_text_search.py

```diff
@@ -171,6 +218,8 @@ def create_romanized_name_fields(romanize_method, **kwargs):
                 appengine_search.TextField(
```

Comment:
Do you need to keep this field in addition to "for_rank" ones? Didn't you remove it in the code before merging?